### PR TITLE
Refactor logging - all remaining logs

### PR DESCRIPTION
**Pull request by gumb0**
This concludes the conversion of all remaining log statements. We can finally merge into develop after this one.

### CODE REVIEW

File: test/tools/jsontests/vm.cpp [hunk 1]
```diff
@@ -256,53 +247,49 @@ eth::OnOpFunc FakeExtVM::simpleTrace() const
...
-            << " ]";
+        LOG(ext.m_logger) << o.str();
+        LOG(ext.m_logger) << " | " << std::dec << ext.depth << " | " << ext.myAddress << " | #"
+                          << steps << " | " << std::hex << std::setw(4) << std::setfill('0') << pc
+                          << " : " << instructionInfo(inst).name << " | " << std::dec << gas
+                          << " | -" << std::dec << gasCost << " | " << newMemSize << "x32"
+                          << " ]";
 
         /*creates json stack trace*/
-        if (VMTraceChannel::verbosity <= g_logVerbosity)
```
[the remark this answers is not in the document]
Comment (winsvega) [reply test/tools/jsontests/vm.cpp]: ask, @pirapira  he worked with vmtest for the last year

[the remark this answers is not in the document]
Comment (pirapira) [reply test/tools/jsontests/vm.cpp]: I think @gumb0 is right.  `--verbosity` controlling anything except `stderr` is weird.

[the remark this answers is not in the document]
Comment (chfast) [reply test/tools/jsontests/vm.cpp]: I also don't Iike that I have to set verbosity 2 to see vmtrace.

**Comment by chfast** [comment]
Some crashes on macOS.

**Pull request merged**